qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -411,18 +411,34 @@ class WebEngineHistory(browsertab.AbstractHistory):
     def current_idx(self):
         return self._history.currentItemIndex()
 
-    def back(self):
-        self._history.back()
+    def back(self, count):
```

Comment:
I usually look at what's truly independent of the backend, and what might change with future backends (say, if we ever add servo 😆). Stuff which only uses methods from `WebEngineHistory` or `WebKitHistory` (like `back` and `forward` now) go to `AbstractHistory` - everything accessing the backend specific history (`self._history`) goes to `WebKitTab`/`WebEngineTab`.

So here you could have `itemAt` and `goToItem` in `Web{Kit,Engine}History` (but see my comment about naming above), and `back` and `forward` in `AbstractHistory`.